src/test/java/hudson/tasks/test/TestResultExtensionTest.java

```diff
@@ -62,8 +62,7 @@ public void testTrivialRecorder() throws Exception {
         assertNotNull("parent action should have an owner", action.run);
         Object resultObject = action.getResult();
         assertNotNull("we should have a result");
-        assertTrue("result should be an TestResult",
```

Comment:
No apparent reason why this file should have been edited here at all.

Reply:
correct I will restore the previous formatting to help people who have a maximum of 65 characters per line